blockstack/blockstackd.py

```diff
@@ -3804,8 +3804,8 @@ class Program {
             'not_distributed_88a422b7ccaa54e5d8225d5130dc562ffdc57cc54bf274f2b631061ae86d0dc0': 'SM1TP7CNY63KQY7DVRDVPNE1X73ND3PT8JX46971V',
             'not_distributed_8941f1658fc78107ad34cf4d8e96a920a5a3257384188fb21e701c5e6a4b81fb': 'SM2ZP9PPEHXGPAAMYRJNMPVET8WRKBSYY1AZ8SWHP',
             'not_distributed_d6d337469b6283046ed909a03a1f45d862d6c1361d5d0ee56ae660e25866b4c4': 'SM3TJ7J6DR9KFQXQGK9SY7M7AEJT33C8NV639J99C',
-            'Reserved_for_exchange_partner_1': '1111111111111111111114oLvT2', # TODO: waiting...
-            'Reserved_for_exchange_partner_2': '1111111111111111111114oLvT2', # TODO: waiting...
+            'Reserved_for_exchange_partner_1': 'SM17BKV3759FG8H6CSZP2NNE2SPV34MZD0YHHA6AT',
```

Comment:
@jessesoslow can you confirm these remaining placeholder addresses are correctly mapped?

Reply:
@zone117x Looks good

Reply:
It's fine by me if it's fine by @jessesoslow.